.github/workflows/verify_samples.yml

```diff
@@ -0,0 +1,50 @@
+name: Build Samples
+
+on: [push, pull_request]
+
+jobs:
+  linux:
+    name: Ubuntu
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
```

Comment:
"Run actions/checkout@master" in log is quite ugly.

XQEMU CI seems to handle this step better in general:

```
- name: Clone Tree
  uses: actions/checkout@v1
  with:
    fetch-depth: 1
```

Note how this also restricts fetch-depth, which improves CI performance.
It also picks a stable version of the checkout action.

Reply:
Both logs appear the same beyond a misplaced `\n` causing double spacing. There is an argument about `stable` versions so I will change back to v1.

Reply:
> Both logs appear the same beyond a misplaced \n causing double spacing.

Where? I'm not sure I follow.

You seem to imply that the log doesn't respect the `--depth` either (which should probably differ in log - although I did not verify it myself)?

---

I meant the following differences:

1. The step has a different name in XQEMU ("Clone Tree") vs. nxdk CI ("Run actions/checkout@master").
2. Should be more stable in the future (by naming an explicit version)
3. Download less data for XQEMU (only the latest revision, other than downloading the entire history)

Reply:
Misunderstood. I don't see a point in adding a name, I think it's rather clear what the external action is doing.

Reply:
Helps with consistency and hides implementation details (avoids confusing nxdk@v1 with action@v1 for example).
The current name sticks out like a sore thumb.

Reply:
In my opinion it should, it's an important step and I would rather know when a step is using a external workflow instead of internal.

Reply:
Regarding the 3 benefits:
1. Name change: This was not done, but I feel strongly about this.
    We disagree, so some other maintainer should take a look (@XboxDev).
    I think that information-hiding is important, especially for a user-facing portions (the name is shown in web-ui).
    Also we don't explicitly mark other external resources in the step names either. ("chocolatey" for example).
2. Stable revision of workflow: This has been changed. Thanks.
3. Shallow clone: This has been tried, but fails. I'd recommend to drop the `depth-fetch`; explained in https://github.com/XboxDev/nxdk/pull/168#issuecomment-560164656.

Reply:
Depth has been fixed.